magento/pwa-studio
# Removes Now.sh pipeline

## Description

We have AWS deploying on a per-PR basis, so we no longer need the Now.sh pipeline. Our https://veniapwa.com demo site has been redirected to the AWS deployment of `master`.

## Related Issue





Closes #1546.

## Verification Steps


1. Verify that `now.sh`-related code doesn't appear anywhere in the codebase
1. Verify that the `Now` GitHub integration no longer appears on PRs

## Screenshots / Screen Captures (if appropriate)

## Checklist

* I have updated the documentation accordingly, if necessary.
* I have added tests to cover my changes, if necessary.

- [x] Remove the `Magento` team from `Now.sh` website
- [x] @mmansoor-magento to remove `Now` GitHub App from `magento` organization
